client/components/DashHeader.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { PropTypes } from 'prop-types';
+
+const DashHeader = props => (
+  <header>
+    <div className="navbar-fixed">
+      <nav>
+        <div className="container-fluid">
+          <div className="nav-wrapper">
+            <Link to="/" className="brand-logo">ShoutIt</Link>
+            <Link
+              to=""
+              data-activates="mobile-links"
+              className="button-collapse"
+            ><i className="material-icons">menu</i></Link>
+
+            <ul id="nav-mobile" className="right hide-on-med-and-down">
+              <li className="active"><Link to="/dashboard">Dashboard</Link></li>
+              <li><Link to="/" className="white-text">Sign out</Link></li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </div>
+    <div id="mobile-links" className="side-nav fixed">
+      <div className="avatar">
+        <Link to="">
+          <img src="img/avatar.jpg" className="center" alt="" />
```

Comment:
If the `alt` property of your `img` element is going to be empty, then there's no point having it here in the first place?